pyaedt/edb_core/components.py

```diff
@@ -1100,6 +1142,80 @@ def add_port_on_rlc_component(self, component=None):
             self._logger.info("Component {} has been replaced by port".format(component.refdes))
             return True
 
+    @pyaedt_function_handler()
+    def add_rlc_boundary(self, component=None, circuit_type=True):
```

Comment:
Could it help the user if we rename the method as:
```suggestion
    def add_rlc_gap_boundary(self, component=None, circuit_type=True):
```

Reply:
actually this command is used either to define gap boundary or circuit one. It is called by the command reaplce_rlc_by_gap_boundaries